src/features/dataset/DatasetHeader.tsx

```diff
@@ -62,14 +63,14 @@ const DatasetHeader: React.FC<DatasetHeaderProps> = ({
           )}
 
           <div className='text-2xl text-qrinavy-500 font-black group hover:text'>
-            {dataset.meta?.title || dataset.name}
+            {header.name}
           </div>
           {showInfo && (
             <div className='flex mt-3 text-sm'>
               <DatasetInfoItem icon='automationFilled' label='automated' iconClassName='text-qrigreen' />
               <DatasetInfoItem icon='disk' label='59 MB' />
-              <DatasetInfoItem icon='download' label='418 downloads' />
-              <DatasetInfoItem icon='follower' label='130 followers' />
+              <DatasetInfoItem icon='download' label={getLabel(header.downloadCount, 'download')} />
+              <DatasetInfoItem icon='follower' label={getLabel(header.followerCount, 'follower')} />
               <DatasetInfoItem icon='lock' label='private' />
```

Comment:
Currently all datasets are public. Let's reflect that here. Rather than defaulting to 'private', let's default to 'public' & use the 'globe' icon.

```
<DatasetInfoItem icon='globe' label='public' />
```